src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -114,13 +117,46 @@ public void setSendToTis(boolean sendToTis) {
     }
 
     /**
-     * @return {@code true} to continuously retry on error
+     * @return the initial interval retry time that will be used as we graduate to larger numbers between retries
      */
-    public boolean isRetryOnError() {
-        return retryOnError;
```

Comment:
Or I guess we need to make sure we configure the load test spring configuration for load test to 1 before this is merged.